keras/src/backend/torch/core.py

```diff
@@ -658,6 +658,16 @@ def random_seed_dtype():
     return "int32"
 
 
+def remat(func, *args, **kwargs):
```

Comment:
Also add it in the numpy and openvino backends, returning plain function results with no remat (for compatibility)